hil-test/tests/esp_wifi_floats.rs

```diff
@@ -129,6 +129,19 @@ mod tests {
             .start_app_core(
                 unsafe { &mut *core::ptr::addr_of_mut!(APP_CORE_STACK) },
                 move || {
+                    // app core starts with interrupts disabled
+                    unsafe {
+                        esp_hal::xtensa_lx::interrupt::enable_mask(
```

Comment:
On core-0 enabling level 1 interrupts is a side effect of `esp_hal::init`

Reply:
How does esp_hal::init achieve it? Why don't we change `start_app_core` to enable interrupts?

Reply:
It's a side effect of installing the GPIO interrupt handler IIRC

Currently we explicitly disable all interrupt on core-1 - we should define what we do for each core (or if we should try to get them in a similar state). But IMHO tinkering with that is out of scope of this PR

I will create an issue for that (so we don't forget about it and more important not do some random change in an unrelated PR)

Reply:
Issue created #3410